hydra-node/test/Hydra/Chain/Direct/Contract/Commit.hs

```diff
@@ -35,33 +37,31 @@ healthyCommitTx =
   (tx, lookupUTxO)
  where
   lookupUTxO =
-    UTxO.singleton (initialInput, toUTxOContext initialOutput)
+    UTxO.singleton (Fixture.testSeedInput, toUTxOContext initialOutput)
       <> UTxO.singleton healthyCommittedUTxO
       <> registryUTxO scriptRegistry
   tx =
     commitTx
       scriptRegistry
       Fixture.testNetworkId
+      (mkHeadId Fixture.testPolicyId)
       commitParty
       (Just healthyCommittedUTxO)
-      (initialInput, toUTxOContext initialOutput, initialPubKeyHash)
+      (Fixture.testSeedInput, toUTxOContext initialOutput, initialPubKeyHash)
```

Comment:
I was confused of the `testSeedInput` usage here. This `TxIn` is the one used by the `testPolicyId`, while this input should correspond to a hypothetical UTxO produced by the initial transaction.

Should: keep it arbitrarily generated (as before)